data/dual-license-responses.json

```diff
@@ -46,7 +46,11 @@
       954034475,
       966646470,
       966648900,
-      996040374
+      996040374,
+      1064509989,
+      1064524289,
+      1064655585,
+      1072732554
```

Comment:
> 1072732554

That seems to be the comment id of your own question @mchowning , not sure, do we include it to the list as well? 🤔

Reply:
Every comment that has been "processed" gets added here. This prevents the script from re-prompting about how to handle a given comment every time it is run, and instead the script will only look at "new" comments (i.e., comments not on this list) the next time it has been run. In other words, it is basically the source of truth for what comments we have already looked at and recorded any relevant consents (or denials of consent).